app/views/shared/_navbar.html.erb

```diff
@@ -1,14 +1,13 @@
-<div class="container">
-  <div id="shared-navbar">
-    <div class="navbar-header">
-      <a href="#" class=" navbar-brand navbar-left" style="background-color: black;">
-        <%= image_tag "codaisseur-logo-horizontal-outlines-white" %>
-      </a>
-    </div>
-    <ul class="nav nav-pills navbar-right">
-      <li class="<%= current_page?(attend_path) ? " active" : " "%>"><%= link_to "Join Taste of Code", attend_path %></li>
-      <li class="<%= current_page?(incompany_path) ? " active" : " "%>"><%= link_to "In Company", incompany_path %></li>
-      <li class="<%= current_page?(sponsor_path) ? " active" : " "%>"><%= link_to "Host Taste of Code", sponsor_path %></li>
-    </ul>
+<div id="shared-navbar">
+  <div class="navbar-header">
+    <a href="#" class=" navbar-brand navbar-left" style="background-color: black;">
```

Comment:
dont use inline `style` attributes. slap an extra class on it, and style that with CSS